exporter/sematextexporter/es_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sematextexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sematextexporter"
+import (
+	"io"
+	"testing"
+
+	"github.com/olivere/elastic"
```

Comment:
This library is deprecated now

Reply:
The "github.com/olivere/elastic" library?

Reply:
Wow, so what library do we use cause this is what was used in the agent